src/main/groovy/au/ala/org/ws/security/AlaWsSecurityGrailsPluginConfiguration.groovy

```diff
@@ -106,6 +106,14 @@ class AlaWsSecurityGrailsPluginConfiguration {
         authenticator.requiredClaims = jwtProperties.requiredClaims
         authenticator.requiredScopes = jwtProperties.requiredScopes
 
+        authenticator.rolesFromAccessToken = jwtProperties.rolesFromAccessToken
+        if (authenticator.accessTokenRoleClaims = jwtProperties.roleAttributes) {
+            authenticator.accessTokenRoleClaims = jwtProperties.roleAttributes
+        }
```

Comment:
Do you think this is correct?

Reply:
That's wrong, I've fixed. 
When roles are read from the access token then the role claims should be read from `roleAttributes`